avocado/plugins/runner_nrunner.py

```diff
@@ -72,9 +72,9 @@ def _populate_task_logdir(self, base_path, task, statuses, debug=False):
         with open(data_file, 'w') as fp:
             fp.write("{}\n".format(task.output_dir))
 
-    def run_suite(self, job, result, test_suite, variants, timeout=0,
-                  replay_map=None, execution_order=None):
+    def run_suite(self, job, result, test_suite, variants, config):
         summary = set()
+        timeout = config.get('run.job_timeout')
         if timeout > 0:
```

Comment:
If we need to check `timeout is not None` in my previous comment, why not here too? If we don't need to check `None` in my previous comment, this is fine.